### DISCUSSION TITLE
Allow TechnoTypes Immune during Chrono Warp stage

### DISCUSSION THREAD

**Discussion by deathreaperz** [Phobos-developers/Phobos#1042] [Ideas and feedback]
### First check

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search and read the issue list to find a similar issue and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

### Description

Allow Chrono techs have immunity/untargetable during Chrono Warp cooldown stage

### INI code

```ini
[General]
WarpImmune=no ;bool

[TechnoType]
Warp.Immune=no ;bool, def to [General]>WarpImmune
```

**Comment by Daiki9527** [comment]
check: https://modenc.renegadeprojects.com/Warpable

Warpable=no

**Comment by Daiki9527** [comment]
plus ArmorTypes to disable targeting
